src/convert.jl

```diff
@@ -202,6 +202,26 @@ function trixi2vtk(filename::AbstractString...;
             verbose && println("| | Element variable: $label...")
             @timeit label vtk_celldata[label] = variable
           end
+
+          # Add node variables
+          for (label, variable) in node_variables
+            verbose && println("| | Node variable: $label...")
+            if reinterpolate
+              if index == 1
```

Comment:
Why is this warning triggered by the `index`? Wouldn't it more sense to trigger it from the corresponding `label`?

Reply:
I agree with you to trigger the warning just for the label `:limiting_coefficient`. Unfortunately, this will be an issue when using it for MCL where we have labels like `Symbol("alpha_", variables[v])` for all variables v, `:alpha_pressure` and `:alpha_entropy`. Is it acceptable to add an if clause like `if contains("limiting", label) || contains("alpha", label)`?

Moreover, the if clause `if index == 1` is used to print the warning only once per call of `trixi2vtk`. Otherwise, for every chosen file, we get a warning. I could move this warning outside the for loop over all files to just print it once.

Reply:
Since Trixi2Vtk.jl is not going to win any code beauty contest, I'd be OK with having this special case for alpha and the limiting coefficient. Although I'd prefer to use ```julia
label == "limiting_coefficient" || startswith(label, "alpha_")
```
instead.

Regarding repeated warnings: just use a sentinel variable (`has_warned_about_interpolation = false`) that you set to `true` after the first warning and then don't warn anymore 🤷‍♂️

Reply:
Done in [aeb6802](https://github.com/trixi-framework/Trixi2Vtk.jl/pull/67/commits/aeb6802cd7a8c58db19b8fc3f85afa6ec686ca9e)